src/backend/distributed/operations/citus_tools.c

```diff
@@ -122,7 +231,8 @@ master_run_on_worker(PG_FUNCTION_ARGS)
 	}
 
 	int commandCount = ParseCommandParameters(fcinfo, &nodeNameArray, &nodePortArray,
-											  &commandStringArray, &parallelExecution);
+											  &commandStringArray,
```

Comment:
is it possible to either (a) undo these unrelated changes (b) have them in a separate commit? Mostly asking for a cleaner git history purposes.

Reply:
I removed these changes. I do not think that I should update these lines at all.

It should be fine now.